### PR TITLE
Minor drape fixes

### PR DESCRIPTION
**Pull request by biodranik**
Pre-requisites for HB

### CODE REVIEW

File: base/internal/message.hpp [hunk 1]
```diff
@@ -24,7 +24,7 @@
 //@{
 template <typename T> inline std::string DebugPrint(T const & t);
 
-std::string DebugPrint(std::string const & t);
+std::string DebugPrint(std::string && t);
```
Comment (vng) [inline base/internal/message.hpp:27]: Why?
```
string s;
DebugPrint(s); //< Will be called DebugPrint(T const & t) overload with streams
```

Comment (biodranik) [reply base/internal/message.hpp]: Right, returned also the original version.

**Comment by biodranik** [comment]
@vng PTAL

File: base/internal/message.hpp [hunk 2]
```diff
@@ -24,7 +24,8 @@
 //@{
 template <typename T> inline std::string DebugPrint(T const & t);
 
-std::string DebugPrint(std::string const & t);
+inline std::string DebugPrint(std::string const & t) { return t; }
+inline std::string DebugPrint(std::string && t) { return std::move(t); }
```
Comment (vng) [inline base/internal/message.hpp:28]: If you worry about it, this is enough
```inline std::string DebugPrint(std::string t) { return t; }```

**Review by vng — APPROVED** [review]

File: base/internal/message.hpp (hunk 2 above)
Comment (biodranik) [reply base/internal/message.hpp]: @vng Right, agree. Fixed.

File: base/internal/message.cpp [hunk 3]
```diff
@@ -1,36 +1,7 @@
 #include "message.hpp"
 
-#include "std/target_os.hpp"
-
 #include <utf8/unchecked.h>
 
-std::string DebugPrint(std::string const & t)
-{
-#ifdef OMIM_OS_WINDOWS
```
Comment (biodranik) [inline base/internal/message.cpp:9]: @Osyotr hope that 2010 year workaround is not needed anymore? Are utf8 strings printed fine now?

Comment (Osyotr) [reply base/internal/message.cpp:9]: I've also removed it so should be fine.

**Pull request merged**